examples/README.md

```diff
@@ -0,0 +1,8 @@
+# Examples
+Example package contains a collection of snap examples and [videos](https://github.com/intelsdi-x/snap/blob/master/examples/videos.md) that demonstrate the snap libraries and usages.
+
+* [configs](https://github.com/intelsdi-x/snap/tree/master/examples/configs) is an example of the global configuration file that powers your plugins
+* [influxdb-grafana]() has an example of [snap Influxdb publisher plugin](https://github.com/intelsdi-x/snap-plugin-publisher-influxdb). The example demonstrates
+Intel PCM (Performance Counter Metrics) publishing to Influxdb and using grafana to view the results. 
```

Comment:
PCM is Performance Counter Monitor: https://software.intel.com/en-us/articles/intel-performance-counter-monitor.